src/classes/CW_CampaignRewardCompControllerTest.cls

```diff
@@ -0,0 +1,260 @@
+/**
+* File Name   :    CW_CampaignRewardCompController.cls
+* Description :    B-109691 - Reward Summary Table 
+*				   that are linked to the campaign 
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 03/04/2014 Cristina P		Initial
+*/
+@isTest
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.